linera-views/src/value_splitting.rs

```diff
@@ -378,7 +412,11 @@ impl TestMemoryStoreInternal {
             cache_size: 1000,
         };
```

Comment:
Corrected. Yes, we are no longer using guards. The use of guards is now mostly internal.
It caused some problem with constructor since I had a `new_from_guard` and similar, and we prefer to have as few constructors as possible.